apps/controllerx/devices/ikea.py

```diff
@@ -30,6 +31,7 @@ def get_z2m_actions_mapping(self):
     def get_deconz_actions_mapping(self):
         return {
             1002: Light.TOGGLE,
+            1001: Light.ON_FULL_BRIGHTNESS,
```

Comment:
I guess this is the event when holding, but does the 1002 fires when holding? Same as z2m?